content/2022/10/26/crypto-1-history-and-basics/index.md

```diff
@@ -0,0 +1,178 @@
+* The age of modern computing, with its ever-increasing amount of computational power (quantum computing, anyone?) poses new problems and threats to proven approaches.
+
+"But where does he want to go with this?", you may want to ask. At this point, cryptography is something that concerns us all. It's in our phones, our banks, our computers and even in our refrigerators and ceiling lamps (hopefully). And I think it's important to understand what's going on. To a degree at least. And to understand where we stand and where we're going, we need to understand where we come from. If you take anything from this paragraph, let it be "the enemy knows the system" - never trust complexity as a security device.
+
+And that's not even all there is to it! If we broaden the picture and take a look at the field of crypto**logy**, we'll find all kinds of useful applications. For example:
+* Defending against attackers (cryptography, which this post mainly focuses on)
+* Attacking secure systems (cryptanalysis, which aims to overcome cryptographic security measures by analyzing them and finding useful weaknesses)
+* Security protocols, which provide a secure link between two interacting parties
+
+### Notation/Ground rules
+Before we begin, we need to agree on a few things. The first of which is "how do we actually write this down?". To keep things consistent across two authors and a number of posts, we'll define the most important details right at the start.
+
+#### Names
+Examples in the realm of cryptography always rely on names to illustrate who is interacting with whom. The most commonly used names are:
+
+**Alice**
+Alice is our "person A". She either pens the original message that starts a (hopefully encrypted) conversation or is the one that states our general problem.
+
+**Bob**
+Similarly, Bob is "person B" and usually the one to interact with Alice.
+
+**Eve**
+Eve is an **e**avesdropper that tries to listen in on whatever it is Alice and Bob are discussing.
+
+**Mallory**
+Mallory is an attacker that relies on **m**alleable algorithms. Don't worry, we'll discuss this when the time comes.
+
+**Oscar**
+Oscar is our generic **o**pponent and, similar to Eve, tries to interfere with the communication between Alice and Bob. He might occasionally show up for the sake of flavour.
+
+We may at times need to introduce additional parties. For the sake of simplicity, all attackers will use names that start with the latter that suits their attack vector best (so a second eavesdropper might be called Eric and a second generic opponent might be called Olivia). All other actors will draw from the remainder of the alphabet.
+
+#### Notation
+When talking about cryptography, especially when doing to in theory, there is not really a way around using math one way or another. We will try to focus on "concept first". This means that we'll introduce math as necessary, explaining new symbols as they come into play instead of building a giant glossary in this post.
```

Comment:
> especially when doing **_to_** in theory

especially when doing **_so_** in theory?

Reply:
```suggestion
When talking about cryptography, especially when doing so in theory, there is not really a way around using math one way or another. We will try to focus on "concept first". This means that we'll introduce math as necessary, explaining new symbols as they come into play instead of building a giant glossary in this post.
```

Makes sense.